### PR TITLE
Improved the way how deployment index is gathered from csv and the patching procedure

### PR DESCRIPTION
**Pull request by rebtoor**
```
❯ export OPERATOR_NAME="dataplane-operator"

❯ OPERATOR_INDEX=$(oc get csv openstack-operator.v0.0.1 -o json | jq -r '.spec.install.spec.deployments | map(.name == $ENV.OPERATOR_NAME + "-controller-manager") | index(true)')

❯ echo $OPERATOR_INDEX                                                                                                                                                         
6

❯ oc patch csv openstack-operator.v0.0.1 --type json \
  -p "[{ "op": "remove", "path": "/spec/install/spec/deployments/${OPERATOR_INDEX}" }]"
clusterserviceversion.operators.coreos.com/openstack-operator.v0.0.1 patched
```

### CODE REVIEW

File: running_local_operator.md [hunk 1]
```diff
@@ -16,15 +16,21 @@ will resolve its CR.
...
-oc get csv openstack-operator.v0.0.1 -o json \
-  | jq -r '.spec.install.spec.deployments[].name'
+export OPERATOR_NAME="openstack-ansiblee-operator"
+OPERATOR_INDEX=(oc get csv openstack-operator.v0.0.1 -o json | jq -r '.spec.install.spec.deployments | map(.name == $ENV.OPERATOR_NAME + "-controller-manager") | index(true)'
 ```
 Update the CSV
 ```
 oc patch csv openstack-baremetal-operator.v0.0.1 --type json \
-  -p='[{"op": "remove", "path": "/spec/install/spec/deployments/<index>"}]'
+  -p="[{"op": "remove", "path": "/spec/install/spec/deployments/${OPERATOR_INDEX}"}]"
```
Comment (stuggi) [inline running_local_operator.md:28]: will this work correct if the outer and inner quotes are `"`?

Comment (rebtoor) [reply running_local_operator.md:28]: yup, I posted a working example in the description of the PR :)

Comment (abays) [reply running_local_operator.md:28]: Not differentiating the quotes makes it slightly awkward to parse, but if it works it works

**Review by abays — APPROVED** [review]
/lgtm

File: running_local_operator.md (hunk 1 above)
Comment (rebtoor) [reply running_local_operator.md:28]: Yeah, you're right @abays ...another approach could be splitting the yaml with the single quotes to parse `${OPERATOR_INDEX}` as a shell var but IMHO it's even worse. :man_shrugging: 
```oc patch csv openstack-operator.v0.0.1 --type json \
  -p '[{ "op": "remove", "path": "/spec/install/spec/deployments/'${OPERATOR_INDEX}'" }]'
```

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**